src/victory-primitives/path-helpers.js

```diff
@@ -1,3 +1,4 @@
+/*eslint no-magic-numbers: ["error", { "ignore": [0, 1, 2, 2.5, 3] }]*/
```

Comment:
I think we want to first look at what to do for base values in eslint configs in the archetype and then what to do for certain source files. My general theory

- Demo, Test: Whatever, disables are fine
- Source: We want to make developers actually think about the magic numbers they're using. We've got tons of issues like `14` (which is a point font that isn't intuitively that and should be coded at the top). For things like "angles" and divisors, etc could be per-file, or in base config. 

... at any rate, good topic for another day 😉